tests/rkt_run_pod_manifest_test.go

```diff
@@ -120,6 +120,28 @@ func TestPodManifest(t *testing.T) {
 			"",
 		},
 		{
+			// Working directory.
+			[]imagePatch{
+				{"rkt-test-run-pod-manifest-working-directory.aci", []string{}},
+			},
+			&schema.PodManifest{
+				Apps: []schema.RuntimeApp{
+					{
+						Name: baseAppName,
+						App: &types.App{
+							Exec:             []string{"/inspect", "--check-cwd=/dir1", "--print-msg=Check-cwd"},
```

Comment:
`--check-cwd` flag seems weird to me, we should do it like the other tests (i.e. it should be `--print-cwd` or something like that)